res_handler_jq_transform.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/mitchellh/mapstructure"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+)
+
+type ResponseTransformJQOptions struct {
+	//FlushInterval time.Duration
+}
+
+type ResponseTransformJQMiddleware struct {
+	Spec   *APISpec
+	config ResponseTransformJQOptions
```

Comment:
If the options are empty, remove it, the type and the code to decode it.

Reply:
I don't understand what I need to do here

Reply:
This field is never used. It can be removed, as well as the mapstructure stuff that decodes into it, and the type that defines `ResponseTransformJQOptions`.